core/src/i18n/en.yaml

```
@@ -117,7 +119,65 @@ discovery-and-network:
 functions:
   title: Functions
   variable:
-    tooltip: This variable comes from a Resource. To get the value, follow the link.
+    buttons:
+      add: Add environment variable
+      open:
+        custom: Custom Variable
+        secret: Secret Variable
+        config-map: Config Map Variable
+    errors:
+      empty: Variable is empty.
+      duplicated: Duplicated variable name.
+      invalid: Variable name must contain alphanumeric characters, can contain '_', no blank spaces.
+      invalid-config: Select the config map name and key
+      invalid-secret: Select the secret name and key
+      restricted: This variable name is restricted and cannot be used. Try a different one.
+    form:
+      config-map: $t(functions.variable.type.config-map)
+      key: $t(functions.variable.header.key)
+      name: Name
+      secret: $t(functions.variable.type.secret)
+      value: $t(functions.variable.header.value)
+    header:
+      key: Key
+      name: Variable Name
+      source: Source
+      value: Value
+    not-found: This Function doesn't have any environment variables yet.
+    not-match: Couldn't find environment variables matching this query.
+    title:
+      create-modal:
+        custom: Create Custom Variable
+        secret: Create Secret Variable
+        config-map: Create Config Map Variable
+      edit-modal:
+        custom: Edit Custom Variable
+        secret: Edit Secret Variable
+        config-map: Edit Config Map Variable
+      environment-variables: Environment variables
+      injected-variables: Environment variables injected via Service Bindings
+    placeholders:
+      name: Variable name
+      value: Variable value
+    popup-error: At least one variable has an incorrect name format, is restricted, duplicated, or empty.
+    tooltip:
+      text: This variable comes from a Resource. To get the value, follow the link.
+      config-map: This variable comes from the "{resourceName}" Config Map.
+      custom: This variable was provided by the user.
+      secret: This variable comes from the "{resourceName}" Secret.
+      service-binding: This variable was injected by the Service Binding referencing Service Instance "{serviceInstanceName}".
+    type:
+      config-map: Config Map
+      custom: Custom
+      secret: Secret
+      service-binding: Service Binding
+    warnings:
+      text: Warning
+      variable-can-override-sbu: This variable can override or be overridden by a variable injected by one of the created Service Bindings.
+      sbu-can-be-override:
```

```suggestion
      sbu-can-be-overridden:
```
`can override` or `can be overridden`